src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int MAX_INT = 25;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final String[] FIGURES = {"Rectangle",
+            "RightTriangle",
+            "Circle",
+            "Square",
+            "IsoscelesTrapezoid"};
+
+    public Figure getRandomFigure() {
+        Figure figure = null;
+
+        switch (FIGURES[random.nextInt(FIGURES.length)]) {
+            case "Rectangle": {
+                figure = getRectangle();
+                break;
```

Comment:
```suggestion
                return getRectangle();
```
the same for other case blocks

Reply:
Case blocks are chenged